src/ui/server/controllers/insurance/account/sign-in/enter-code/index.test.ts

```diff
@@ -261,6 +261,16 @@ describe('controllers/insurance/account/sign-in/enter-code', () => {
         });
       });
 
+      describe('when req.session.submittedData.insuranceEligibility is an empty object', () => {
```

Comment:
is there a test to check it goes to the right page (problem with service) if empty object?